Category query

OleanjiKingCode opened the pull request:
# Randomizing categories shown in homepage

Using the array filter method and math . random I removed one random category from the category list on reload 

![image](https://user-images.githubusercontent.com/75235148/198338348-031404bd-7ab4-4f67-8b46-394ab5c4203f.png)

![image](https://user-images.githubusercontent.com/75235148/198338430-d90e7a7b-81ad-45aa-adad-3bc653ad4483.png)

![image](https://user-images.githubusercontent.com/75235148/198338511-3e4052d6-fd2c-4d0f-97ef-161fbe84af5f.png)

kesar commented on src/components/Landing/CategoriesList.tsx:
```diff
@@ -18,6 +18,8 @@ interface CategoriesListProps {
 }
 const CategoriesList = ({ categories }: CategoriesListProps) => {
   const { t } = useTranslation()
+  const RandcategoryItem: string =
```
variable names are in camelCase

I think its gonna be more readable if you just:

const filteredCategories =  categories.splice(Math.floor(Math.random()*categories.length), 1);

and use that w current map

kesar replied:
you haven solve this or reply the reason

OleanjiKingCode commented:
I didn't use the splice method due to an error I had while using It, I decided to just go with the one I initially used

kesar commented:
> I didn't use the splice method due to an error I had while using It, I decided to just go with the one I initially used

that's not a proper response to a code review. Please explain what error or why do you think your way its cleaner / better to the suggested one.

OleanjiKingCode commented:
@kesar ?

kesar reviewed (approved):
looking good

kesar commented:
wait

https://github.com/EveripediaNetwork/ep-ui/blob/main/src/data/AllCategoriesData.ts#L73

u are renaming a file

OleanjiKingCode commented:
> wait
> 
> https://github.com/EveripediaNetwork/ep-ui/blob/main/src/data/AllCategoriesData.ts#L73
> 
> u are renaming a file
Yes, I changed that because the link used to display the dapps image in prod doesn't work in dev so I changed the it now it should show in both dev and prod

kesar commented:
but then you need to rename it in the file i sent you, otherwise we will have a 404 image ther

kesar commented:
good 👍🏻
